lib/helpers/enumHelper.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:App/data_classes/recipe.dart';
+
+/// Returns the value of the enum type as string
+String getStringFromEnum(enumType) {
+  return enumType.toString().split('.')[1];
```

Comment:
```suggestion
  return describeEnum(enumtype);
```
kan hent verdier direkte

Reply:
Euuuu :P sexy! good catch